samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/DeliveryMessages.properties

```
@@ -625,3 +625,4 @@ seb_download_configuration=Download Configuration
 seb_download_seb=Download Safe Exam Browser
 seb_launch_seb=Launch Safe Exam Browser
 seb_quit_browser=Quit Safe Exam Browser
+seb_preview_info=Assessments that are configured to use Safe Exam Browser can not be previewed.
```

@mpellicer Is that correct?
```suggestion
seb_preview_info=Las examenes configuradas para utilizar Safe Exam Browser no se pueden previsualizar.
```